includes/template-library/sources/local.php

```diff
@@ -1292,10 +1292,31 @@ public function maybe_render_blank_state( $which ) {
 
 			$current_type = 'template';
 
-			$inline_style .= '#elementor-template-library-tabs-wrapper {display: none;}';
+			$args['additional_inline_style'] = '#elementor-template-library-tabs-wrapper {display: none;}';
 		}
 
+		$this->render_blank_state( $current_type, $args );
+	}
+
+	public function render_blank_state( $current_type, $args = [] ) {
 		$current_type_label = $this->get_template_label_by_type( $current_type );
+		$inline_style = '#posts-filter .wp-list-table, #posts-filter .tablenav.top, .tablenav.bottom .actions, .wrap .subsubsub { display:none;}';
+
+		if ( ! empty( $args['additional_inline_style'] ) ) {
```

Comment:
Use the `wp_parse_args` approach like here: https://github.com/elementor/elementor/blob/36947060eaaa85394b991f70d6cabfe4465269ad/includes/template-library/sources/local.php#L453-L459
and don't check every argument if it's empty.

Reply:
Handled.